### PR TITLE
Add BIOSSES dataset

### PR DESCRIPTION
**Pull request by bwang482**
Adding the biomedical semantic sentence similarity dataset, BIOSSES, listed in "Biomedical Datasets - BigScience Workshop 2021"

### CODE REVIEW

File: datasets/biosses/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+  - en-US
```
Comment (lhoestq) [inline datasets/biosses/README.md:7]:
```suggestion
- en
```

File: datasets/biosses/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ 'sentence 2': 'Not surprisingly, GATA2 knockdown in KRAS mutant cells resulted in a striking reduction of active GTP-bound RHO proteins, including the downstream ROCK kinase'
+ 'score': 2.2}
+```
+
+
+### Data Fields
+
+- `sentence 1`
+- `sentence 2`
+- `score`
```
Comment (lhoestq) [inline datasets/biosses/README.md:95]:
```suggestion
- `sentence 1`: string
- `sentence 2`: string
- `score`: float ranging from 0 (no relation) to 4 (equivalent)
```

File: datasets/swedish_medical_ner/README.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
+---
```
Comment (lhoestq) [inline datasets/swedish_medical_ner/README.md:1]: It looks like this PR contains changes about another dataset that BIOSSES, could you remove these files ?
We can add `swedish_medical_ner` in #2873

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this dataset :)
The dataset card looks good. Same for the dataset script :)

I just noticed that there were files that are unrelated to BIOSSES, but rather about the `swedish_medical_ner` datasset. Can you remove these files please ?
Feel free to ping me when you removed the files to merge the PR :)

File: datasets/swedish_medical_ner/README.md (hunk 3 above)
Comment (bwang482) [reply datasets/swedish_medical_ner/README.md]: Thanks @lhoestq ! I have removed the `swedish_medical_ner` folder now.

**Review by lhoestq — APPROVED** [review]
Thanks !

Merging now :)

**Pull request merged**